index.html

```diff
@@ -34,31 +34,244 @@
+                        <a href="#"><img src="./images/Vector.svg" alt="sure" id="sure"></a>
                     </li>
                     <li>
-                        <a href="#"><img src="./images/Vector-2.svg" alt="github"></a>
+                        <a href="#"><img src="./images/Vector-3.svg" alt="twitter" id="twitter"></a>
                     </li>
                     <li>
-                        <a href="#"><img src="./images/Vector.svg" alt="sure"></a>
+                        <a href="#"><img src="./images/Vector-1.svg" alt="M" id="m"></a>
                     </li>
                 </div>
             </div>
+            <div class="action">
+                <button class="btn">Get my resume</button>
+            </div>
+
+            <div class="languages">
+                <h3>Languages</h3>
+                <ul class="lang-tag">
+                    <li>
+                        <span>
+                            <img src="./images/js.png" alt="">
+                            <h3>Javascript</h3>
+                        </span>
+                    </li>
+                    <li>
+                        <span>
+                            <img src="./images/html.png" alt="">
+                            <h3>HTML</h3>
+                        </span>
+                    </li>
+                    <li>
+                        <span>
+                            <img src="./images/css.png" alt="">
+                            <h3>CSS</h3>
+                        </span>
+                    </li>
+                </ul>
+            </div>
+
+            <ul class="others">
+                <a href="#">
+                    <li class="framework"> Framework</li> <i>></i>
+                </a>
+                <hr>
+                <a href="#">
+                    <li class="framework">Skills</li> <i>></i>
+                </a>
+                <hr>
+            </ul>
         </div>
     </section>
-    <!-- ─── End Headline Section ────────────────────────────────────── -->
+    <!-- ─── End About Myself ────────────────────────────────────────── -->
+
+    <!-- ─── Contact Me ──────────────────────────────────────────────── -->
+    <section class="contact">
+        <div class="contact-info">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="#" class="contact-form">
+            <input type="text" name="name" id="name" placeholder="Cosmos Hagan">
+            <input type="email" name="email" id="email" placeholder="cosmoshagan@gmail.com">
+            <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here"></textarea>
+            <div class="action">
+                <button class="btn">Get In Touch</button>
+            </div>
+        </form>
+    </section>
```

Comment:
- [ ] Please remove this section for now, as it will be implemented in another project in the coming days. Please for all your microverse projects, always implement only what you find in the project requirement for that week. This is how most real-life work is, you do only what is marked out to be completed except otherwise.
![image](https://user-images.githubusercontent.com/84156714/212425581-31bd9d92-6d6c-4838-a8ae-2604ad8fa0c8.png)